src/tools/rbd/action/Bench.cc

```diff
@@ -221,9 +221,14 @@ int do_bench(librbd::Image& image, io_type_t io_type,
   uint64_t i;
   uint64_t start_pos;
 
-  // disturb all thread's offset, used by seq IO
+  float unit_len = (float)size/io_size/io_threads;
```

Comment:
Nit: probably no need for using floating point -- if it's not properly divisible, let it round down / up to the next IO size (otherwise you could still have non-block aligned IO).

Reply:
sounds reasonable. what about this time? just let unit_len be uint64_t to make sure it aligned IO size boundary